bcda/constants/constants.go

```diff
@@ -5,5 +5,9 @@ const SMALLACOUUID = "3461C774-B48F-11E8-96F8-529269fb1459"
 const MEDIUMACOUUID = "C74C008D-42F8-4ED9-BF88-CEE659C7F692"
 const LARGEACOUUID = "8D80925A-027E-43DD-8AED-9A501CC4CD91"
 
+const BCDA_FHIR_MAX_RECORDS_EOB_DEFAULT = 200
+const BCDA_FHIR_MAX_RECORDS_PATIENT_DEFAULT = 25000
+const BCDA_FHIR_MAX_RECORDS_COVERAGE_DEFAULT = 15000
```

Comment:
Indeed they can, but I put them in here so we can start keeping track of our constant values. Honestly I would like this file to grow with more and more constants, that way we know where all of our values are stored and changes become that much easier.

Reply:
Keeping all constants together feels like an anti-pattern to me-- the file could grow pretty large and would end up being difficult to maintain, and naming has to be very specific to avoid clashes. I'd strongly prefer to store them where they are used.

Reply:
the anti-pattern argument is a good one 👍 . Okay, so do we want to navigate away from a single constants file completely or only store constant values that are truly global in there?

Reply:
An argument could probably be made for either of those. :) Getting rid of that file altogether seems out of scope for this ticket-- a topic to revisit later?